rt/rs/microprofile-client/src/main/java/org/apache/cxf/microprofile/client/MicroProfileClientProviderFactory.java

```diff
@@ -91,18 +91,14 @@ protected void setProviders(boolean custom, boolean busGlobal, Object... provide
 
     public List<ResponseExceptionMapper<?>> createResponseExceptionMapper(Message m, Class<?> paramType) {
 
-        List<ResponseExceptionMapper<?>> candidates = new LinkedList<>();
-
-        for (ProviderInfo<ResponseExceptionMapper<?>> em : responseExceptionMappers) {
-            if (handleMapper(em, paramType, m, ResponseExceptionMapper.class, true)) {
-                candidates.add(em.getProvider());
-            }
-        }
-        if (candidates.size() == 0) {
+        if (responseExceptionMappers.isEmpty()) {
```

Comment:
They should never actually be empty, since we require the default one to be registered.

Reply:
The only time where it would be empty would be if the user set the `microprofile.rest.client.disable.default.mapper` property to true.  In that case we can avoid the sort code.

Reply:
Ah yes, forgot about the property.